src/templates/en/2019/base_chapter.html

```diff
@@ -17,9 +17,33 @@
 indexBoxTitle.addEventListener('click', function(e) {
   indexBox.classList.toggle('show');
 });
+
+fetch(`https://discuss.httparchive.org/t/{{ metadata.get('discuss') }}.json`)
+    .then(function(r) { return r.json(); })
+    .then(function(r) {
+  if (!r) {
+    return;
+  }
+
+  var comments = +r.posts_count - 1;
+  if (isNaN(comments)) {
+    return;
+  }
+  var el = document.getElementById('num_comments');
+  el.innerText = comments + ' ' + (comments == 1 ? 'comment' : 'comments');
+});
 </script>
 {% endblock %}
 
+{# Calls to action for readers who want to engage more with this chapter. #}
+{% macro render_actions() %}
+  <div class="discuss">
+    <a class="alt btn" href="https://discuss.httparchive.org/t/{{ metadata.get('discuss') }}">
```

Comment:
Open in new tab so you can leave the chapter you want to discuss still open?

```suggestion
    <a class="alt btn" target="_blank" href="https://discuss.httparchive.org/t/{{ metadata.get('discuss') }}">
```

Reply:
I've heard of some [arguments like these](https://css-tricks.com/use-target_blank/) that target=_blank is usually not a good idea. Our audience is tech-savvy so I trust them to Cmd+Click or use the back button if they want to retain the chapter page.